synfig-studio/src/gui/widgets/widget_link.h

```diff
@@ -40,18 +42,46 @@ namespace studio {
 
 class Widget_Link: public Gtk::ToggleButton
 {
-	synfig::String tooltip_inactive_;
-	synfig::String tooltip_active_;
+	std::string tooltip_inactive_;
+	std::string tooltip_active_;
```

Comment:
Any reason for keeping these redundant string variable? I guess property_* variables do their jobs.

Reply:
`Widget_Link(const std::string &tlt_inactive, const std::string &tlt_active);`

https://github.com/synfig/synfig/blob/eafeda4828fea262646abba906b6650fb16591f4/synfig-studio/src/gui/renddesc.cpp#L596-L597

Do you want me to update those two legacy codes too?

Reply:
There is no problem with the constructor.
Why do we have two variables that store the same data? Either we use `property_tooltip_inactive` or `tooltip_inactive_` . ;)
Or am I misreading it ?

Reply:
I kept that legacy constructor because the user supplied unique string arguments in `gui/renddesc.cpp` file

Will update the file to use only glib custom property then :)

Reply:
Now I remember... I thought `Widget_Link` would have to be loaded from glade file for custom glib property to work, that's why I kept those string variables intact. How silly... yet revealing how lacking I am in Glib area.

Learned so much from you @rodolforg. Thanks for the opportunity and the guidance 🙇🏽‍♂️ 😄